### ISSUE TITLE
Theme preferences not respected

### ISSUE THREAD

**Issue by ChildishGiant** [akiraux/Akira#651]
## Expected Behavior
Akira should use the light/dark theme based on the user's choice, although that can be overridden in the settings still.

## Current Behavior
The user's preference isn't reflected at all. It appears this functionality has been attempted but it's not working https://github.com/akiraux/Akira/blob/83c7b03b1a29274d46a4811c94ff4a8be4e050dd/src/Window.vala#L93

## Possible Solution
https://docs.elementary.io/develop/apis/color-scheme#using-the-users-preference

## Screenshots of the Problem
 

https://user-images.githubusercontent.com/13716824/129961388-3ca39b27-9446-430a-a196-8375498528f8.mp4

## Your Environment (for bugs)

```
Linux elementary-os-desktop-39fa46e2 5.11.0-25-generic #27~20.04.1-Ubuntu SMP Tue Jul 13 17:41:23 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
NAME="elementary OS"
VERSION="6 Odin"
ID=elementary
ID_LIKE=ubuntu
PRETTY_NAME="elementary OS 6 Odin"
LOGO=distributor-logo
VERSION_ID="6"
HOME_URL="https://elementary.io/"
DOCUMENTATION_URL="https://elementary.io/docs/learning-the-basics"
SUPPORT_URL="https://elementary.io/support"
BUG_REPORT_URL="https://github.com/elementary/os/issues/new"
PRIVACY_POLICY_URL="https://elementary.io/privacy-policy"
VERSION_CODENAME=odin
UBUNTU_CODENAME=focal
```
- Akira Version: v0.0.16 - alpha (appcenter)

**Comment by Alecaddd** [comment]
Sure, I'll implement this before next release, thanks for the report.

**Comment by Abdallah-Moh** [comment]
@Alecaddd This should be closed after #657
